### ISSUE TITLE
Allow for vector-scalar multiplication in History fields

### ISSUE THREAD

**Issue by mmanyin** [GEOS-ESM/MAPL#1959]
In a History collection that is output on a lat/lon grid,   the vectors U and V  will be properly regridded from the cube sphere to lat/lon  if the expression   U;V   is used.

I'm looking for a way to output a field such as  U*O3,  where U is a vector from DYN, and O3 is a scalar from one of the Chemistry Gridded Components.  Currently ‘U*O3’ will treat U and O3 as scalars, and therefore not regrid to lat/lon properly.

The syntax could be along these lines:   ‘(U*O3);(V*O3)’   or   ‘(U;V)*O3’ .

Note that we considered outputting U;V and O3 separately, and then multiplying as a post-processing step, but if the collection is time-averaged, particularly daily or monthly, the answer will be markedly different from multiplying at each timestep.

**Comment by mathomp4** [comment]
Assigning @bena-nasa since he's who I think of for this, but I also know @darianboggs is looking at History, so maybe this is something he could work on?

**Comment by tclune** [comment]
Yes - unless @bena-nasa sees an easy way to do this in MAPL2, it should wait for MAPL3.   Then we will have proper tangent vectors available and  it should be easier to extend the expression mechanism.

@bena-nasa One immediate question:   Do you think the expression processing would go in the top History gridcomp?  Or would it go in the collection gridcomp?
